### PR TITLE
Conceptcra 1878 multithreaded mogelijk m

### PR DESCRIPTION
**Pull request by johnnyborg**

### CODE REVIEW

**Comment by integratedforpublishers** [comment]
Can one of the admins verify this patch?

File: Command/IndexerRunCommand.php [hunk 1]
```diff
@@ -141,10 +198,57 @@ private function runExternal(InputInterface $input, OutputInterface $output)
...
+            }
+
+            if ($input->getOption('blocking')) {
+                $output->writeln('Running in blocking mode, waiting until all started process are done');
+
+                // While the pool contains processes we're running
+                while ($pool->count()) {
+                    foreach ($pool as $i => $process) {
+                        if (!$process->isRunning()) {
+                            $output->writeln(sprintf('Process %d finnished.', ($i+1)));
```
Comment (gerjanvdbosch) [inline Command/IndexerRunCommand.php]: Typo: finished

**Comment by johnnyborg** [comment]
@integratedforpublishers @gerjanvdbosch fixed the comments.

File: Command/IndexerRunCommand.php [hunk 2]
```diff
@@ -58,43 +117,44 @@ protected function configure()
...
-            $indexer = $this->getContainer()->get('integrated_solr.indexer');
-
-            if ($output->isDebug() && method_exists($indexer, 'setDebug')) {
-                $indexer->setDebug();
+            if ($output->isDebug() && method_exists($this->indexer, 'setDebug')) {
+                $this->indexer->setDebug();
             }
 
-            $indexer->execute();
+            $this->indexer->execute();
```
Comment (gerjanvdbosch) [inline Command/IndexerRunCommand.php:167]: Add EntityManager and DocumentManager clear (memory usage).

**Pull request merged**